.github/pull_request_template.md

```diff
@@ -15,7 +15,9 @@ resolves #
 
 ### Checklist
 
+- [ ] I have read [the contributing guide](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md) and understand what's expected of me
 - [ ] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
```

Comment:
nit: It bothers me that we have this checkbox but also have an action that validates this.  To me the action feels like enough.  Same for the changelog one.  If a GHA exists to check this and posts a message to the PR when it fails, a checkbox here is redundant as long as the comment that gets posted provides enough context.

Reply:
I could see leaving the `Skip Changelog` entry out of this, since that's really just for us (fixing READMEs, GHAs, or test cases). I'd expect 99% of community-contributed PRs to want changelog entries

Reply:
I agree that our team should be the only ones making changes that don't involve a changelog.  Additionally, we like giving our community recognition for their contributions to dbt-core and a changelog is how we do that.